packages/rspack/src/builtin-loader/lightningcss/index.ts

```diff
@@ -0,0 +1,153 @@
+};
+
+export function browserslistToTargets(
+	browserslist: string[]
+): Record<string, number> {
+	const targets: Record<string, number> = {};
+	for (const browser of browserslist) {
+		const [name, v] = browser.split(" ");
+		if (BROWSER_MAPPING[name] === null) {
+			continue;
+		}
+
+		const version = parseVersion(v);
+		if (version == null) {
+			continue;
+		}
+
+		if (targets[name] == null || version < targets[name]) {
+			targets[name] = version;
+		}
+	}
+
+	return targets;
+}
+
+function parseVersion(version: string) {
+	const [major, minor = 0, patch = 0] = version
+		.split("-")[0]
+		.split(".")
+		.map(v => Number.parseInt(v, 10));
+
+	if (isNaN(major) || isNaN(minor) || isNaN(patch)) {
+		return null;
+	}
+
+	return (major << 16) | (minor << 8) | patch;
+}
+
+export enum Features {
+	Empty = /*													*/ 0,
+	Nesting = /*												*/ 1 << 0,
+	NotSelectorList = /*								*/ 1 << 1,
+	DirSelector = /*										*/ 1 << 2,
+	LangSelectorList = /*								*/ 1 << 3,
+	IsSelector = /*											*/ 1 << 4,
+	TextDecorationThicknessPercent = /*	*/ 1 << 5,
+	MediaIntervalSyntax = /*						*/ 1 << 6,
+	MediaRangeSyntax = /*								*/ 1 << 7,
+	CustomMediaQueries = /*							*/ 1 << 8,
+	ClampFunction = /*									*/ 1 << 9,
+	ColorFunction = /*									*/ 1 << 10,
+	OklabColors = /*										*/ 1 << 11,
+	LabColors = /* 											*/ 1 << 12,
+	P3Colors = /*												*/ 1 << 13,
+	HexAlphaColors = /*									*/ 1 << 14,
+	SpaceSeparatedColorNotation = /*	 	*/ 1 << 15,
+	FontFamilySystemUi = /*							*/ 1 << 16,
+	DoublePositionGradients = /*				*/ 1 << 17,
+	VendorPrefixes = /*									*/ 1 << 18,
+	LogicalProperties = /*							*/ 1 << 19,
+	Selectors = Features.Nesting |
+		Features.NotSelectorList |
+		Features.DirSelector |
+		Features.LangSelectorList |
+		Features.IsSelector,
+	MediaQueries = Features.MediaIntervalSyntax |
+		Features.MediaRangeSyntax |
+		Features.CustomMediaQueries,
+	Color = Features.ColorFunction |
+		Features.OklabColors |
+		Features.LabColors |
+		Features.P3Colors |
+		Features.HexAlphaColors |
+		Features.SpaceSeparatedColorNotation
+}
+
+export interface Targets {
+	android?: number;
+	chrome?: number;
+	edge?: number;
+	firefox?: number;
+	ie?: number;
+	ios_saf?: number;
+	opera?: number;
+	safari?: number;
+	samsung?: number;
+}
+
+export interface Drafts {
+	/** Whether to enable @custom-media rules. */
+	customMedia?: boolean;
+}
+
+export interface NonStandard {
+	/** Whether to enable the non-standard >>> and /deep/ selector combinators used by Angular and Vue. */
+	deepSelectorCombinator?: boolean;
+}
+
+export interface PseudoClasses {
+	hover?: string;
+	active?: string;
+	focus?: string;
+	focusVisible?: string;
+	focusWithin?: string;
+}
+
+export type LoaderOptions = {
+	errorRecovery?: boolean;
+	targets?: Targets | string[] | string;
+	include?: Features;
+	exclude?: Features;
+	draft?: Drafts;
+	nonStandard?: NonStandard;
+	pseudoClasses?: PseudoClasses;
+	unusedSymbols?: Set<String>;
```

Comment:
Can we use `string[]`? it is more common to use

```suggestion
	unusedSymbols?: string[]
```

Reply:
here is because lightningcss uses Set<string>

Reply:
I prefer Array, because it is easier to use and can align with `LightningCssMinimizerRspackPlugin`

Reply:
Changed to Array